packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.ts

```diff
@@ -85,6 +90,19 @@ export abstract class XYChartComponent extends TimeseriesChartComponent
         }
     }
 
+    public onPrimaryDescClick(legendSeries: IChartAssistSeries<IAccessors>) {
+        this.eventBus.getStream(INTERACTION).next({
+            payload: {
+                data: legendSeries,
+                interactionType: TimeseriesInteractionType.Series,
+            },
+        });
+    }
+
+    public onSecondaryDescClick(legendSeries: IChartAssistSeries<IAccessors>) {
+        this.chartAssist.toggleSeries(legendSeries.id, this.chartAssist.isSeriesHidden(legendSeries.id));
```

Comment:
It seem like if we conditionally remove the `stopPropagation` from the secondaryDescription element in the case where there's no link that the nui-legend-series should handle the series toggling for you. Can you give that a try instead?

Reply:
See previous comment. I still think this is the case.